MSSQL/OMOPLoader.sql

```diff
@@ -110,19 +110,19 @@ CREATE TABLE [i2o_config_modifier]  (
 	[c_path]         	varchar(400) NULL,
 	[c_target_column]	varchar(50) NULL 
 	)
-INSERT INTO [COVID_OMOP_Mart].[dbo].[i2o_config_modifier]([c_domain], [c_tablename], [c_path], [c_target_column])
+INSERT INTO [AllOfUs_Mart].[dbo].[i2o_config_modifier]([c_domain], [c_tablename], [c_path], [c_target_column])

```

Comment:
Ah, I see I mistakenly pushed COVID_OMOP_Mart! These should just be the table name, no db or schema. Can you fix that?

Reply:
Removed the database and schema references.